one/AppDelegate.swift

```diff
@@ -27,20 +27,29 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         window = UIWindow(frame: UIScreen.mainScreen().bounds)
         
-        let viewController = ViewController()
-        let navigationController = UINavigationController(rootViewController: viewController)
         let globeIcon = UIImage(named: "tabbar-feed")?.imageWithRenderingMode(UIImageRenderingMode.AlwaysOriginal)
-        let globeItem: UITabBarItem = UITabBarItem.init(title: "", image: globeIcon, tag: 1)
+        let globeItem: UITabBarItem = UITabBarItem.init(title: "", image: globeIcon, tag: 0)
         globeItem.imageInsets = UIEdgeInsetsMake(-10, 0, 10, 0)
-        navigationController.tabBarItem = globeItem
-        let viewControllers = [navigationController]
-        let tabBarController = UITabBarController()
-        tabBarController.tabBar.shadowImage = UIImage()
         
+        let getCareIcon = UIImage(named: "tabbar-GetCare")?.imageWithRenderingMode(UIImageRenderingMode.AlwaysOriginal)
+        let getCareItem = UITabBarItem.init(title: "", image: getCareIcon, tag: 1)
+        getCareItem.imageInsets = UIEdgeInsetsMake(-10, 0, 10, 0)
+        
+        let chartIcon = UIImage(named: "tabbar-user")?.imageWithRenderingMode(UIImageRenderingMode.AlwaysOriginal)
+        let chartItem = UITabBarItem.init(title: "", image: chartIcon, tag: 2)
+        chartItem.imageInsets = UIEdgeInsetsMake(-10, 0, 10, 0)
```

Comment:
Let's make these insets a constant so that we can reuse them for other `UITabBarItem`s